src/host/getset.cpp

```diff
@@ -357,6 +359,17 @@ void ApiRoutines::GetNumberOfConsoleMouseButtonsImpl(ULONG& buttons) noexcept
             WI_ClearFlag(gci.Flags, CONSOLE_USE_PRIVATE_FLAGS);
         }
 
+        const auto newQuickEditMode{ WI_IsFlagSet(gci.Flags, CONSOLE_QUICK_EDIT_MODE) };

```

Comment:
I'd almost add a comment with a reference to "GH#9970" here, because I doubt that a future reader would know why the heck QE has to be disabled to enable mouse mode. This thread has become the canonical source of truth on the matter, so it'll be a helpful reference for any future readers.